frontend/pendingTranslations.ftl

```diff
@@ -35,3 +35,7 @@ profile-label-set-your-custom-domain-free-user = Get your own email domain with
 
 tooltip-email-domain-explanation-title-free = Get your own { -brand-name-relay } email domain
 tooltip-email-domain-explanation-part-one-free = With { -brand-name-premium }, you can create unlimited, custom { -brand-name-relay } masks on the go using a unique { -brand-name-relay } email domain — you won’t even have to generate them here first.
+
+nav-profile-manage-fxa-v2 = Manage your account
```

Comment:
suggestion (non-blocking): Should we drop `fxa` from this string ID too?
```suggestion
nav-profile-manage-account = Manage your account
```

Reply:
done